bin/znapzend

```diff
@@ -107,6 +109,22 @@ Examples:
 
 Define the log level when logging to file. Default is debug.
 
+=item B<--mailto>={I<admin@domain.com>}
+
+send warning- and/or error-level logs to this address
+
+Examples:
+
+ --mailto=admin@domain.com
+
+=item B<--mailprog>={I<filepath>}
+
+path to program to use for sending mail
+
+Examples:
+
+ --mailprog=/usr/bin/mail
+
```

Comment:
I guess you should mention that the code expects that the mail program accepts the `-s` argument